balancer/main.go

```diff
@@ -0,0 +1,56 @@
+package balancer
+
+import (
+	"context"
+	"net/http"
+	"time"
+
+	"github.com/R-a-dio/valkyrie/balancer/current"
+	"github.com/R-a-dio/valkyrie/config"
+	"github.com/R-a-dio/valkyrie/errors"
+	"github.com/R-a-dio/valkyrie/storage"
+)
+
+// Execute executes the balancer with the context ctx and config cfg.
+// Execution of the balancer can be halted by cancelling ctx.
+func Execute(ctx context.Context, cfg config.Config) error {
+	const op errors.Op = "balancer/Execute"
+
+	br, err := NewBalancer(ctx, cfg)
+	if err != nil {
+		return errors.E(op, err)
+	}
+	return errors.E(op, br.start(ctx))
```

Comment:
don't return errors that are not errors, `errors.E(op, nil) != nil` check the actual error return before wrapping please